src/components/dialogs/field-hook.js

```diff
@@ -460,17 +460,19 @@ export const useEquipmentTableValues = ({
                 }
                 let objects = Object.keys(csvData).map(function (key) {
                     return {
-                        equipmentID: csvData[key][0].trim(),
+                        equipmentID: isGeneratorOrLoad
```

Comment:
What justifies keeping side blanks when generator or load ?

Reply:
yes, there is no difference, i changed